apps/src/templates/courseOverview/CourseOverview.js

```diff
@@ -233,19 +219,6 @@ class CourseOverview extends Component {
               : descriptionTeacher
           }
         />
-        <div>
```

Comment:
Are we sure this isn't needed any more... I am thinking in particular if a teacher gets to a curriculum through the catalog, we would still want this bar at the top... I could be wrong, but wanted to double check

Reply:
The only thing in this top row is currently behind a flag and it's the single section assigner. The CourseOverviewActionRow (above) contains a multiple section assign button ([link](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/courseOverview/CourseOverviewActionRow.tsx#L99))